src/tt_store.erl

```diff
@@ -1,9 +1,94 @@
 -module(tt_store).
 
--export([find_by_time/2]).
+-behaviour(gen_server).
+-define(SERVER, ?MODULE).
 
--record(talk, {title :: string(), start_time :: calendar:datetime(), end_time :: calendar:datetime(), location :: string()}).
+-include_lib("stdlib/include/ms_transform.hrl").
+%% API
+-export([start_link/0, add/4, find_by_time/2, list/0, find_by_name/1]).
 
--spec(find_by_time(calendar:datetime(), calendar:datetime()) -> [#talk{}]).
-find_by_time(_StartTime, _EndTime) ->
-  [#talk{title = "Title", start_time = calendar:local_time(), end_time = calendar:local_time(), location =  "Location"}].
+%% Supervisor callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2,
+         terminate/2, code_change/3]).
+
+%% DB record
+-record(talk, {title :: string(), startTime :: {Hour, Minute, Second}, endTime :: {Hour, Minute, Second}, location :: any()}).
```

Comment:
It is better to use the type exported from the [calendar module](http://erlang.org/doc/man/calendar.html) - `calendar:time()` instead of writing `{Hour, Minute, Second}`.